redaxo/src/core/lib/rex.php

```diff
@@ -271,6 +271,16 @@ public static function getServerName()
         return self::getProperty('servername');
     }
 
+    /**
+     * Returns whether the current request is served via https/ssl.
+     *
+     * @return bool true when https/ssl, otherwise false.
+     */
+    public static function isHttps()
```

Comment:
Wäre das nicht besser in `rex_request` aufgehoben?

Reply:
Stimmt